### PR TITLE
Wrote new test for create function

### PR DESCRIPTION
**Pull request by PreciousTosin**

### CODE REVIEW

File: test/feature-speck.js [hunk 1]
```diff
@@ -5,9 +5,12 @@ var note = new NotesApplication("John");
 
 describe("The functionality of the create function", function(){
```
Comment (andela-kakpobome) [inline test/feature-speck.js:6]: This document should be named feature-**spec** not feature-**speck**

File: test/feature-speck.js [hunk 2]
```diff
@@ -5,9 +5,12 @@ var note = new NotesApplication("John");
...
 
-	it("It should return an array", function() {
-		var noteformat = note.create("James is a guy","Matt is a guy")
-		expect(noteformat.length).to.be.above(0);
+	it("It should add a value to the array each time create function is called", function() {
+		var noteformat = []
+		noteformat.push(note.create("James is a guy"));
```
Comment (andela-kakpobome) [inline test/feature-speck.js:10]: Your **create function** should not return values, it should populate the notes collection of the NotesApplication instance

**Comment by PreciousTosin** [comment]
@kesiena. Thanks for the feedback, I made the corrections in the #12 pull request. Kindly look through it for any area where I could make improvements.
Thanks

**Comment by PreciousTosin** [comment]
@kesienaAkp: Thanks for the feedback, I made the corrections in the #12 pull request. Kindly look through it for any area where I could make improvements.
Thanks.

**Pull request merged**